src/component/form-field.js

```diff
@@ -19,6 +21,12 @@ export class FormField {
     this.viewManager = viewManager;
   }
 
+  attached() {
+    if (!existy(this.element.key)) {
```

Comment:
So we hide it elsewhere. The key can't be 0 and any other falsy value will work with @VMBindraban his suggestion. I vote to remove existy and just keep it simple.